app/src/main/java/org/fossasia/openevent/general/auth/EditProfileFragment.kt

```diff
@@ -58,7 +58,7 @@ class EditProfileFragment : Fragment() {
                 context?.let { ctx ->
                     val drawable = AppCompatResources.getDrawable(ctx, R.drawable.ic_account_circle_grey_24dp)
                     drawable?.let { icon ->
-                        if(!imageUrl.equals("")) { // picasso requires the imageUrl to be non empty
+                        if(!imageUrl.isEmpty()) { // picasso requires the imageUrl to be non empty
```

Comment:
It'll fail on null

Reply:
```suggestion
                val userFirstName = it.firstName.nullToEmpty()
                val userLastName = it.lastName.nullToEmpty()
                val imageUrl = it.avatarUrl.nullToEmpty()
                rootView.firstName.setText(userFirstName)
                rootView.lastName.setText(userLastName)
                context?.let { ctx ->
                    val drawable = AppCompatResources.getDrawable(ctx,   drawable.ic_account_circle_grey_24dp)
                    drawable?.let { icon ->
                        if(!imageUrl.isEmpty()) { // picasso requires the imageUrl to be non empty
                            Picasso.get()
                                         .load(imageUrl).....
```
I think ` val imageUrl = it.avatarUrl.nullToEmpty()` will take care of that

Reply:
OK. So, we can add the if statement even above the `context.let`

Reply:
Sure, I will do it

Reply:
@iamareebjamal Done